src/components/caseStudyPreview.js

```diff
@@ -8,10 +8,10 @@ const CaseStudyPreview = ({ caseStudies }) => {
   if (!Array.isArray(caseStudies)) return null
   return (
     <div className="w-full bg-gray-200 p-5">
-      <h1 className="text-center text-2xl pt-5 p-10 md:text-4xl ">
+      <h1 className=" text-gray-600 text-center text-2xl pt-5 p-10 md:text-4xl ">
```

Comment:
If all text in this component is gray-600 then put it on the parent, no point duplicating class names on all the children.